__tests__/topics.test.js

```diff
@@ -0,0 +1,25 @@
+const app = require("../app.js");
+const request = require("supertest");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data/index.js");
+const db = require("../db/connection.js");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => {
+  db.end();
+});
+
+describe("Topics EndPoint", () => {
+  describe("Get: /api/topics", () => {
+    test("Get:200 get all topics", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.topics.length).toBe(3);
```

Comment:
This test is fine, but could be more complete if we checked for expected properties on the returned topic objects